### ISSUE TITLE
Request clearInvalidatedAnswers feature

### ISSUE THREAD

**Issue by budupaul** [surveyjs/survey-library#1811]
### Are you requesting a feature, reporting a bug or asking a question?
Feature

### What is the current behavior?

There should be an option when saving survey results to remove the answers that haven't passed validation and save without them

**Comment by tsv2013** [comment]
Are you looking for the https://surveyjs.io/Documentation/Library/?id=surveymodel#clearIncorrectValues method?

**Comment by budupaul** [comment]
@tsv2013 This doesn't seem to work. I save the survey results every time the page changes. So let's say on my current page I have a validator for a text question, if the answer doesn't pass the validation when I change the page, I want that answer to be deleted and save the answers from the survey without including the invalid answer. This doesn't happen if I call clearIncorrectValues. Hvae I missed something?

**Comment by tsv2013** [comment]
@budupaul  Yes, the "clearIncorrectValues" is more related to the inaccessible choice items.
We'll keep your suggestion in mind.
As a workaround you can validate all questions before save and remove invalid answers by your code.
